src/components/cc-token-api-update-form/cc-token-api-update-form.types.d.ts

```diff
@@ -0,0 +1,31 @@
+export type TokenApiUpdateFormState =
+  | TokenApiUpdateFormStateLoading
+  | TokenApiUpdateFormStateError
+  | TokenApiUpdateFormStateLoaded
+  | TokenApiUpdateFormStateUpdating;
+
+export interface TokenApiUpdateFormStateLoading {
+  type: 'loading';
+}
+
+export interface TokenApiUpdateFormStateError {
+  type: 'error';
+}
+
+export interface TokenApiUpdateFormStateLoaded {
+  type: 'loaded';
+  values: {
```

Comment:
suggestion: maybe we could extract the `values` object to its proper type. This way it could be used directly in the `CcTokenChangePayload` type. It would also allows to type the `SKELETON_VALUES` in the component.